_delphi_utils_python/delphi_utils/export.py

```diff
@@ -113,10 +113,11 @@ def create_export_csv(
             "geo_id",
             "val",
             "se",
+            "direction",
```

Comment:
question: Do you know how not having `direction` as an expected column so far hasn't been a problem? I actually don't know what `direction` describes at all, or if/how it shows up in our database. 
Additionally, how does adding `direction` as an expected column affect other indicators that don't have this column in their current implementation (nchs_mortality, etc. indicators that aren't claims_hosp or doctor_visits essentially)?

Reply:
it's because doctor-visits and hospital claims were using their own functions instead of create_export_csv. The other indicators don't use direction

Reply:
have the same essential question about how it wasn't needed before

my understanding of `direction` is that it was meant to be a classification of which direction the indicator is trending: up, down, steady.

as for your last question, i tested the pandas filter function and it doesn't complain if you have extra columns in there, so for instance
```py
df = pd.DataFrame({"a": [0, 1], "c": [1, 2]})
df.filter(items=["a", "b"])
```
just returns the df with the "a" column back.

Reply:
@dshemetov 's understanding is correct, but "direction" is now very deprecated...  [It doesnt even exist in the database and is just stubbed out as null](https://github.com/cmu-delphi/delphi-epidata/blob/4e130d26c6c6fa393eae94d5cc31a29460d1897e/src/ddl/v4_schema.sql#L102) in case some user is still depending on it.  We should be removing instances of it wherever we can instead of adding new ones.